invenio_theme/assets/semantic-ui/less/invenio_theme/theme/collections/menu.overrides

```diff
@@ -61,6 +61,10 @@
       border-radius: 0;
       background-color: transparent;
 
+      .ui.dropdown.custom-plus-menu {
```

Comment:
Can you double check if this is needed, maybe it's enough by using the already created classes `rel-b{alias}-{value}` ?